src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -18,32 +18,43 @@
  */
 package ch.njol.skript.expressions;
 
-import ch.njol.util.VectorMath;
-import org.bukkit.Location;
-import org.bukkit.event.Event;
-
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
-import ch.njol.util.Kleenean;
+import ch.njol.skript.Skript;
 import ch.njol.util.coll.CollectionUtils;
+import ch.njol.util.Kleenean;
+import ch.njol.util.VectorMath;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.bukkit.entity.Player;
+import org.bukkit.Location;
 import org.bukkit.util.Vector;
 
 @Name("Yaw / Pitch")
-@Description("The yaw or pitch of a location or vector.")
-@Examples({"log \"%player%: %location of player%, %player's yaw%, %player's pitch%\" to \"playerlocs.log\"",
+@Description("The yaw or pitch of an entity, location or vector.")
+@Examples({
+	"log \"%player%: %location of player%, %player's yaw%, %player's pitch%\" to \"playerlocs.log\"",
 	"set {_yaw} to yaw of player",
-	"set {_p} to pitch of target entity"})
-@Since("2.0, 2.2-dev28 (vector yaw/pitch)")
+	"set {_p} to pitch of target entity",
+	"set yaw of all players to 0"
+})
+@Since("2.0, 2.2-dev28 (vector yaw/pitch), INSERT VERSION (change support for entities)")
+@RequiredPlugins("Minecraft 1.13 (change support for non-player entities), Paper 1.19 (change support for players)")
 public class ExprYawPitch extends SimplePropertyExpression<Object, Number> {
 
+	static final boolean PLAYER_SUPPORT;
+
 	static {
-		register(ExprYawPitch.class, Number.class, "(0¦yaw|1¦pitch)", "locations/vectors");
+		PLAYER_SUPPORT = Skript.methodExists(Player.class, "setRotation", float.class, float.class);
+		register(ExprYawPitch.class, Number.class, "(0¦yaw|1¦pitch)", "entities/locations/vectors");
```

Comment:
```suggestion
		register(ExprYawPitch.class, Number.class, "(:yaw|pitch)", "entities/locations/vectors");
```
Then use `usesYaw = parseResult.hasTag("yaw")`